src/buildCommon.js

```diff
@@ -318,8 +318,9 @@ const makeSpan = function(
     classes?: string[],
     children?: DomChildNode[],
     options?: Options,
+    style?: {[name: string]: string},
```

Comment:
Is it worth defining a flow type for this string-to-string map?

Reply:
Also, you should probably update the comment to mention this argument (and maybe also explain when to use styles instead of classes?)

Reply:
I think I'm to change this to `cssStyle` to differentiate from LaTeX's concept of styles.  I'll add a comment, but basically this should only be used by dynamic styles which we can't define a class for.